contracts/stellar-axelar-gateway/src/auth.rs

```diff
@@ -158,11 +125,7 @@ fn message_hash_to_sign(env: &Env, signers_hash: BytesN<32>, data_hash: &BytesN<
 }
 
 fn update_rotation_timestamp(env: &Env, enforce_rotation_delay: bool) -> Result<(), ContractError> {
-    let last_rotation_timestamp: u64 = env
-        .storage()
-        .instance()
-        .get(&DataKey::LastRotationTimestamp)
-        .unwrap_or(0);
+    let last_rotation_timestamp: u64 = storage::last_rotation_timestamp(env).unwrap_or(0);
```

Comment:
```suggestion
    let last_rotation_timestamp = storage::last_rotation_timestamp(env).unwrap_or(0);
```
Type hint shouldnt be needed now